client/my-sites/backup/controller.js

```diff
@@ -25,6 +26,12 @@ import WpcomBackupUpsellPlaceholder from './wpcom-backup-upsell-placeholder';
 
 const debug = new Debug( 'calypso:my-sites:backup:controller' );
 
+// Helper function to create Calypso notice handlers
```

Comment:
Changes in this file are mostly to pass the success/error notice handlers used on Calypso to the FileBrowser context provider.